public-ui/src/app/components/db-no-results/no-results-search.component.tsx

```diff
@@ -0,0 +1,188 @@
+import { Component, Input } from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { domainToRoute, surveyIdToRoute } from 'app/utils/constants';
+import { reactStyles } from 'app/utils/index';
+import { navigateByUrl } from 'app/utils/navigation';
+import { LoadingDots } from 'app/utils/spinner';
+import { environment } from 'environments/environment';
+import { Configuration, DataBrowserApi } from 'publicGenerated/fetch';
+import * as React from 'react';
+
+const api = new DataBrowserApi(new Configuration({ basePath: environment.publicApiUrl }));
+
+const styles = reactStyles({
+    loadingText: {
+        fontFamily: 'GothamBook, Arial, sans-serif',
+        fontWeight: 'normal',
+        fontStyle: 'normal',
+        fontStretch: 'normal',
+        lineHeight: '1.5',
+        fontSize: '16px',
+        letterSpacing: 'normal',
+        textAlign: 'left',
+        color: '#262262',
+        paddingLeft: '0'
+  },
+  noResults: {
+    marginTop: '-1rem',
+    padding: '1em'
+  },
+  loadingDiv: {
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'flex-start',
+      alignItems: 'center',
+      marginBottom: '1em'
+  },
+  spinnerDiv: {
+      marginTop: '1em',
+      marginBottom: '0.2em',
+      marginLeft: '0.5em'
+  }
+});
+
+const styleCss =
+`
+a:link,a:visited,a{
+    color:#2aa3d8;
+}
+`;
+
+interface Props {
+    searchValue: string;
+    measurementTestFilter: number;
+    measurementOrderFilter: number;
+    domainMatch: Function;
+}
+
+interface State {
+    domainInfoResults: Array<any>;
+    surveyModuleResults: Array<any>;
+    pmResults: Array<any>;
+    fitbitResults: Array<any>;
+    loading: boolean;
+}
+
+export const NoResultSearchComponent = (class extends React.Component<Props, State> {
+    constructor(props) {
+        super(props);
+        this.handleOnClick = this.handleOnClick.bind(this);
```

Comment:
Since you're calling `handleClick` with an arrow function in the template, it will take care of binding to `this`, so we can remove this line.

Reply:
Thanks changed this.